bootstrap.sh

```diff
@@ -5,18 +5,40 @@ cd $(dirname "$BASH_SOURCE")
 set -ev
 
 # Update puppet to version >= 3.2.2 before using puppet provisioning.
-wget https://apt.puppetlabs.com/puppetlabs-release-pc1-trusty.deb
-dpkg -i puppetlabs-release-pc1-trusty.deb
+if [ ! -e puppetlabs-release-pc1-trusty.deb ]
```

Comment:
There are two issues here, redirection and re-download. I just had a second look at https://github.com/sgmap/mes-aides-ops/commit/30faef27df70c59442d241d53aa934c59ec5f616 and will amend it to `curl --location --remote-name`.